modules: add platformOptimizations (#172)
fufexan/nix-gaming: README.md

}
```

## Modules

> Here are some NixOS modules for settings gaming related options.

Author: ```suggestion
Here are some NixOS modules for setting gaming related options.
```

You can calculate the theoretical latency by dividing `quantum` by `rate`
(`48/48000` is exactly 1ms).

### SteamOS sysctl settings

> [SteamOS](https://store.steampowered.com/steamos) on the steam deck has set some
> specific sysctl settings. These include settings to enable some games to run at all,
> or perform better.

Author: ```suggestion
[SteamOS](https://store.steampowered.com/steamos) on the steam deck has set some
specific sysctl settings, so that some games can be run at all, or perform better under 
certain circumstances.
```

> This module extends the Steam module from Nixpkgs but can be set as a standalone
> by just enabling them.

Author: ```suggestion
This module extends the Steam module from Nixpkgs but can be enabled as a 
standalone option.
```
